src/components/Sidebar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import {
+  ChartPieIcon,
+  UserGroupIcon,
+  UsersIcon,
+  GlobeAltIcon,
+  InboxInIcon,
+  CalendarIcon,
+  FolderIcon,
+  AdjustmentsIcon,
+  SupportIcon,
+} from '@heroicons/react/solid';
+import { CogIcon } from '@heroicons/react/outline';
+
+const Sidebar = ({ style }: { style: string }) => {
+  return (
+    <div
+      className={`${style} flex-col fixed h-[100%] pt-16 bg-white dark:bg-dark-bg border-r p-4`}
+    >
+      <div className="list-none pr-8">
+        <li className="mb-4 hover:text-primary">
+          <NavLink to={'to'} className="flex flex-row">
+            <ChartPieIcon className="w-5 mr-2 dark:text-dark-text-fill" />
+            <span className="text-base dark:text-dark-text-fill">
+              Dashboard
+            </span>
+          </NavLink>
+        </li>
+        <li className="mb-4 hover:text-primary">
+          <NavLink to={'to'} className="flex flex-row">
+            <UserGroupIcon className="w-5 mr-2 dark:text-dark-text-fill" />
+            <span className="text-base dark:text-dark-text-fill">
+              Organizations
+            </span>
+          </NavLink>
+        </li>
+        <li className="mb-4 hover:text-primary">
+          <NavLink to={'to'} className="flex flex-row">
+            <UsersIcon className="w-5 mr-2 dark:text-dark-text-fill" />
+            <span className="text-base dark:text-dark-text-fill">Admins</span>
+          </NavLink>
+        </li>
+        <li className="mb-4 hover:text-primary">
+          <NavLink to={'to'} className="flex flex-row">
```

Comment:
For now since we dont have these pages, I would suggest going for the **`#link`** tag to disable navigation